http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -237,6 +272,26 @@ protected void resolveUrls() {
         }
     }
 
+    private JsonWebKeySet createrealmKeySet() throws Exception{
```

Comment:
"realm" is terminology specific to Keycloak so I don't think we should use it in the method name here.

This method is meant to obtain the encryption keys from the OpenID provider, right? We should rename this method to reflect that.

Reply:
removed the function from this class and moved it to a utility class. New function is called extractPublicKeySet